sound/soc/intel/common/soc-acpi-intel-bxt-match.c

```diff
@@ -50,6 +84,7 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_bxt_machines[] = {
 	{
 		.id = "INT34C3",
 		.drv_name = "bxt_tdf8532",
+		.machine_quirk = apl_quirk,
 		.sof_fw_filename = "intel/sof-apl.ri",
 		.sof_tplg_filename = "intel/sof-apl-tdf8532.tplg",
 		.asoc_plat_name = "0000:00:0e.0",
```

Comment:
so we are planning to add another "INT34C3" entry with different machine_quirk for LeafHill?

Reply:
no, there is a single entry and the quirk determines if this machine descriptor is used as is or replaced by another entry from another table. In the case of LeafHill this codec is not present so it's just discarded.